Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
```

Comment:
### [event_sourcery](https://github.com/envato/event_sourcery)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.13.0 → 0.22.0

[_[change-log](https://github.com/envato/event_sourcery/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/event_sourcery)_]





<details>
<summary>Commits</summary>

A change of **188** commits. See the full changes on [the compare page](https://github.com/envato/event_sourcery/compare/v0.13.0...v0.22.0).

These are the first 10 commits:
- (9b61bae) [Be generic about the error reporting service one can use](https://github.com/envato/event_sourcery/commit/9b61baebf6107490174c1a00dae075a37064fa68)
- (57b130a) [Document `EventSourcery.configure`](https://github.com/envato/event_sourcery/commit/57b130aaca48405605bd08c55283bbad8e285be3)
- (e31e52b) [annotating all the private methods in the dirs im doing](https://github.com/envato/event_sourcery/commit/e31e52bc4b7412c13b29c7bdb9aeb413457e3317)
- (1f07e49) [esp_process yard doc](https://github.com/envato/event_sourcery/commit/1f07e4963d2e0ef52f782c27a9ab30e4640e8cfa)
- (00a402e) [adding in doc for start! in esp_runner](https://github.com/envato/event_sourcery/commit/00a402edb83fed3bda76077897998aef407cd4be)
- (91c95fe) [documenting the event_stream_processor_registry](https://github.com/envato/event_sourcery/commit/91c95fecfb5ccbdf1b70423a8947bfa5bdff11d1)
- (0b7d8c9) [typo](https://github.com/envato/event_sourcery/commit/0b7d8c94b88c7e182fa316480b85953427c72651)
- (cbaaf15) [remove all the privates not needed](https://github.com/envato/event_sourcery/commit/cbaaf15ee4a26a10674899007d458257899c9e6e)
- (858acd0) [Address Steves feedback](https://github.com/envato/event_sourcery/commit/858acd009da7254a11b7503a708aedf7ce63a4ae)
- (5e38f90) [Document Config#logger](https://github.com/envato/event_sourcery/commit/5e38f902973a5abf46d55baecb539755a2c2e69c)


</details>